CommentUpdater/src/main/java/org/eupathdb/sitesearch/data/comments/CommentUpdater.java

```diff
@@ -58,15 +63,23 @@
   private final DatabaseInstance _commentDb;
 
   private final String _commentSchema;
+  
+  private final CommentSolrDocumentFields _docFields;
+  
+  private final CommentUpdaterSql _updaterSql;
 
   public CommentUpdater(
     final String           solrUrl,
     final DatabaseInstance commentDb,
-    final String           commentSchema
+    final String           commentSchema,
+    final CommentSolrDocumentFields docFields,
+    final CommentUpdaterSql updaterSql
   ) {
     _solrUrl = solrUrl;
     _commentDb = commentDb;
-    _commentSchema = commentSchema;
+    _commentSchema = commentSchema + (commentSchema.endsWith(".") ? "" : ".");
```

Comment:
Use DBPlatform.normalizeSchema() here to add the . and check a few other things.